evm_arithmetization/src/cpu/kernel/tests/add11.rs

```diff
@@ -208,6 +210,7 @@ fn test_add11_yml() {
 
 #[test]
 fn test_add11_yml_with_exception() {
+    init_logger();
```

Comment:
for debugging purposes? I'm not against, though wondering why it was added (and why only this one and not the one above)